recipe/meta.yaml

```diff
@@ -691,12 +888,15 @@ outputs:
         - pyarrow.cuda     # [cuda_compiler_version != "None" and not win]
       commands:
         # libraries that depend on python (and hence aren't in libarrow itself)
-        - test -f ${SP_DIR}/pyarrow/libarrow_python.so                            # [linux]
-        - test -f ${SP_DIR}/pyarrow/libarrow_python_flight.so                     # [linux]
-        - test -f ${SP_DIR}/pyarrow/libarrow_python.dylib                         # [osx]
-        - test -f ${SP_DIR}/pyarrow/libarrow_python_flight.dylib                  # [osx]
-        - if not exist %SP_DIR%\pyarrow\arrow_python.dll exit 1                   # [win]
-        - if not exist %SP_DIR%\pyarrow\arrow_python_flight.dll exit 1            # [win]
+        - test -f ${SP_DIR}/pyarrow/libarrow_python.so                              # [linux]
+        - test -f ${SP_DIR}/pyarrow/libarrow_python_flight.so                       # [linux]
+        - test -f ${SP_DIR}/pyarrow/libarrow_python_parquet_encryption.so           # [linux]
+        - test -f ${SP_DIR}/pyarrow/libarrow_python.dylib                           # [osx]
+        - test -f ${SP_DIR}/pyarrow/libarrow_python_flight.dylib                    # [osx]
+        - test -f ${SP_DIR}/pyarrow/libarrow_python_parquet_encryption.dylib        # [osx]
+        - if not exist %SP_DIR%\pyarrow\arrow_python.dll exit 1                     # [win]
+        - if not exist %SP_DIR%\pyarrow\arrow_python_flight.dll exit 1              # [win]
+        - if not exist %SP_DIR%\pyarrow\arrow_python_parquet_encryption.dll exit 1  # [win]
```

Comment:
Reduce to libraries that aren't already in `pyarrow` (and adapt comment)

Reply:
At the moment all `pyarrow-core`, `pyarrow` and `pyarrow-all` contain `libarrow_python`, `libarrow_python_flight` and `libarrow_python_parquet_encryption`. I've removed the test commands for `pyarrow-all` as the files are present on the rest. We probably should remove them for `pyarrow` as they are tested on `pyarrow-core` or we might want to create an `install-pyarrow.sh` script that picks the specific libraries for each one but I am not sure if the effort is worth it.
To be honest the big libraries are the Arrow CPP ones, the ones that we could filter on pyarrow (`arrow_python_flight` and `arrow_python_parquet_encryption`) on 15.0.0 are some KB size:
```
$ ls -lhrt libarrow*so
-rwxrwxr-x 1 raulcd raulcd 115K feb  5 15:50 libarrow_python_flight.so
-rwxrwxr-x 1 raulcd raulcd  37K feb  5 15:50 libarrow_python_parquet_encryption.so
-rwxrwxr-x 1 raulcd raulcd 2,6M feb  5 15:50 libarrow_python.so
```